docs/rfc-2-vending-machine.adoc

```diff
@@ -95,11 +95,17 @@ The vending machine contracts are not upgradeable.
 * T token address: TBD
 * NU token address: `0x4fE83213D56308330EC302a8BD641f1d0113A4Cc`
 * KEEP token address: `0x85Eee30c52B0b379b046Fb0F85F4f3Dc3009aFEC`
+* NU Staking Escrow address: TBD
```

Comment:
@cygnusv, could you provide this address for me? I'll update the line accordingly.

Reply:
StakingEscrow: `0xbbD3C0C794F40c4f993B03F65343aCC6fcfCb2e2`
And since T token and Vending Machines are already up on mainnet, do you mind updating those as well?

* T token contract address: 0xCdF7028ceAB81fA0C6971208e83fa7872994beE5
* NU<>T vending machine contract address: 0x1CCA7E410eE41739792eA0A24e00349Dd247680e
* KEEP<>T vending machine contract address: 0xE47c80e8c23f6B4A1aE41c34837a0599D5D16bb0

Reply:
Thanks, updated, see f7142bf.